src/main/java/org/folio/circulation/domain/CreateRequestService.java

```diff
@@ -58,15 +66,54 @@ private static RequestAndRelatedRecords setRequestQueuePosition(
     }
   }
 
+  private static HttpResult<RequestAndRelatedRecords> refuseWhenRequestCannotBeFulfilled(
+    RequestAndRelatedRecords requestAndRelatedRecords) {
+
+    RequestPolicy requestPolicy = requestAndRelatedRecords.getRequestPolicy();
+    RequestType requestType =  requestAndRelatedRecords.getRequest().getRequestType();
+
+    if(!requestPolicy.allowsType(requestType)) {
+      return failed(failure(
+        requestType.getValue() + " requests are not allowed for this patron and item combination", "requestType",
```

Comment:
Do you think we should make "requestType" a public static in `Request` so that this doesn't become a magic string? I doubt that we'll change the JSON field name and the only 2 uses are probably here and in `Request`, so it is probably not a big deal, but I thought I'd mention it.

Reply:
It's a good idea to make it public, and I did, but I think we may need to have a general strategy when referencing these JSON fields in general.